handlers/main.yml

```diff
@@ -0,0 +1,60 @@
+---
+- name: Download and unpack latest syslinux
+  unarchive:
+    remote_src: yes
+    src: https://mirrors.edge.kernel.org/pub/linux/utils/boot/syslinux/Testing/{{ syslinux_latest.stdout }}/syslinux-{{ syslinux_latest.stdout }}-pre1.tar.gz
+    dest: "{{ tftpd_directory }}"
+    owner: "{{ tftpd_owner }}"
+    group: "{{ tftpd_group }}"
+    mode: "{{ tftpd_mode }}"
+  listen: syslinux_new_version
```

Comment:
Wouldn't it be possible to refactor the handlers to a block? That way, you would write `listen` only once. Not sure this is possible here though.

Reply:
nope, I think I tried to use blocks in handlers and ansible complied about it